apps/mobile/src/features/approver/components/fee-card.tsx

```diff
@@ -52,31 +50,39 @@ export function FeeCard({ feeType, amount }: FeeCardProps) {
         </Text>
         <FeeBadge type="normal" />
       </Box>
-      <Pressable flexDirection="row" onPress={() => {}} py="3">
-        <Flag
-          img={
-            <Avatar bg="ink.background-secondary" p="1">
+
+      <Box mx="-5">
+        <Cell.Root pressable={true} onPress={() => {}}>
```

Comment:
Maybe this shouldn't be pressable with no `onPress` defined? 🤔

Reply:
Not sure what's the intent there, I just moved over the existing logic. Perhaps this is still being worked on by Edgar or Fara.

Reply:
@edgarkhanzadian is working on this.

Reply:
👍 now we have a release out, albeit a beta one we should tighten up on stuff like this